app/models/rules/basic.rb

```diff
@@ -1,4 +1,4 @@
-class Rules::Basic < Struct.new(:auction)
+class Rules::Basic < Rules::BaseRules
```

Comment:
Going to defer this to a later PR, where we'll also move some files around in the view directories